CLEO4.vcxproj

```diff
@@ -65,21 +65,21 @@
     <IgnoreWarnCompileDuplicatedFilename>true</IgnoreWarnCompileDuplicatedFilename>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>CLEO4</RootNamespace>
-    <WindowsTargetPlatformVersion>7.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release GTASA|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>false</UseDebugLibraries>
     <CharacterSet>MultiByte</CharacterSet>
-    <PlatformToolset>v141_xp</PlatformToolset>

```

Comment:
Is dropping XP in future releases a final decision? XP is unsupported by Microsoft but (rather unfortunately) it still has a user base, and I suspect a large subset of this user base do play GTA SA since that's a game of that age.

I see this as a risk because of scripters not adopting newer CLEO features due to the complaints of the XP user base.

On the other hand, keeping XP support needs more work on the CLEO developers side. We'd probably need a CI server that tests that the project still builds in vs141_xp, and finally upload these build artifacts somewhere.

I'm not favorable to either decisions, just pointing it out in order to ensure it is a conscious one.

Reply:
I can judge by a number of cleo.li visitors. Share of those who use Windows OS (87% of total visitors) reads as this (stats for the period Oct 2019-Oct 2020):
```
Win 10: 49.9%
Win 7: 30%
Win 8: 5%
Win XP: 0,96%
```

Reply:
Oh, that's great! I guess it shouldn't be a big deal then.

Reply:
I expected my website MixMods, which the main public is Latin countries, to have a higher value due to the large amount of third world countries on statistics, but actually it was only 0.2% in the last year. 61% Windows 10.